src/live-preview.ts

```diff
@@ -394,13 +461,18 @@ export default class LivePreview {
                 this.linkClickHandler
             );
 
-            tooltip.innerHTML = "";
-            tooltip.appendChild(this.tooltipChild.singular);
+            tooltipInnerContainer.innerHTML = "";
+            tooltipInnerContainer.appendChild(this.tooltipChild.singular);
+            tooltip.appendChild(tooltipInnerContainer);
             this.tooltip = tooltip;
         }
         this.updateTooltipPosition();
     };
 
+    private toggleHoveringOnEditButton = (isHoveringOnTooltip: boolean) => {
+        this.isHoveringOnTooltip = isHoveringOnTooltip;
+    };
+
```

Comment:
We can consider moving it to another file.

Reply:
sure, will move this function to utils files